engine/configurator/conf_cyaml.h

```diff
@@ -0,0 +1,297 @@
+ *
+ * Its fields are defined in depends_entries_fields_schema.
+ */
+extern const cyaml_schema_value_t depends_schema;
+
+/**
+ * The object mapping's field definitions schema is an array.
+ *
+ * All the field entries will refer to their parent mapping's structure,
+ * in this case, 'object_type'.
+ */
+extern const cyaml_schema_field_t object_fields_schema[];
+
+/**
+ * The instance mapping's field definitions schema is an array.
+ *
+ * All the field entries will refer to their parent mapping's structure.
+ */
+extern const cyaml_schema_field_t instance_fields_schema[];
+
+/**
+ * The backup_entry mapping's field definitions schema is an array.
+ *
+ * All the field entries will refer to their parent mapping's structure,
+ * in this case, 'backup_entry'.
+ */
+extern const cyaml_schema_field_t backup_entry_fields_schema[];
+
+/**
+ * The value for a backup_entry is a mapping.
+ *
+ * Its fields are defined in backup_entry_fields_schema.
+ */
+extern const cyaml_schema_value_t backup_entry_schema;
+
+/**
+ * The backup mapping's field definitions schema is an array.
+ *
+ * All the field entries will refer to their parent mapping's structure,
+ * in this case, 'backup_seq'.
+ */
+extern const cyaml_schema_field_t backup_fields_schema[];
+
+/**
+ * Top-level schema of the backup. The top level value for the backup is
+ * a sequence.
+ *
+ * Its fields are defined in backup_entry_fields_schema.
+ */
+extern const cyaml_schema_value_t backup_schema;
+
+/**
+ * The schema of object. Its value is a mapping.
+ *
+ * Its fields are defined in object_fields_schema.
+ */
+extern const cyaml_schema_value_t object_schema;
+
+/**
+ * The schema of instance. Its value is a mapping.
+ *
+ * Its fields are defined in instance_fields_schema.
+ */
+extern const cyaml_schema_value_t instance_schema;
+
+/**
+ * The history_entry mapping's field definitions schema is an array.
+ *
+ * All the field entries will refer to their parent mapping's structure,
+ * in this case, 'history_entry'.
+ */
+extern const cyaml_schema_field_t history_entry_fields_schema[];
+
+/**
+ * The value for a history_entry is a mapping.
+ *
+ * Its fields are defined in history_entry_fields_schema.
+ */
+extern const cyaml_schema_value_t history_entry_schema;
+
+/**
+ * The history field definitions schema is an array.
+ *
+ * All the field entries will refer to their parent mapping's structure,
+ * in this case, 'history_seq'.
+ */
+extern const cyaml_schema_field_t history_fields_schema[];
+
+/**
+ * Top-level schema of the history. The top level value for the history is
+ * a sequence.
+ *
+ * Its fields are defined in history_entry_fields_schema.
+ */
+extern const cyaml_schema_value_t history_schema;
+
+/**
+ * Our CYAML config.
+ *
+ * If you don't want to change it between calls, make it const.
+ *
+ * Here we have a very basic config that difined in conf_cyaml.c.
+ */
+extern cyaml_config_t cyaml_config;
+
+/**
+ * Process CYAML errors to TE errors
```

Comment:
There should be a dot at the end of headline. In other places too.

Reply:
Fixed.